README.md

```diff
@@ -45,6 +45,12 @@ listener.on('close', function() {
 listener.close();
 </script>
 ```
+Testing
+-------
+1. Add pulse credentials to test profile in config.yml
+2. Add taskcluster credentials to test profile in config.yml. Ensure you have the necessary scopes.
```

Comment:
It would be nice if this specified either the necessary scopes or a role that can be assumed.